supernode/daemon/mgr/task/manager_util_test.go

```diff
@@ -125,3 +122,81 @@ func (s *TaskUtilTestSuite) TestAddOrUpdateTask(c *check.C) {
 		c.Check(taskInfo, check.DeepEquals, v.task)
 	}
 }
+
+func (s *TaskUtilTestSuite) TestAddDfgetTask(c *check.C) {
```

Comment:
I think it's so simple that it's not necessary to add a unit test with `AddDfgetTask`.And if you want to add a test, maybe it's better to do it at `supernode/mgr/dfgettask`.